mockserver.js

```diff
@@ -135,8 +135,42 @@ app.get('/classdiagram/MULTIPLE_CLASSES', (req, res) => {
 
 // Add class
 app.post('/classdiagram/MULTIPLE_CLASSES/class', (req, res) => {
+  const className = req.body.className;
+  const xPos = req.body.x;
+  const yPos = req.body.y;
+  
+  //TODO: unsure what these ids are for, hard coded for now
+  var valueId = 16;
```

Comment:
@And7ew can you make line 143 and 144 global variables so that it gets updated whenever the POST request is called.